modules/virt-configuring-ip-for-vm.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+[id="virt-configuring-ip-for-vm_{context}"]
+= Configuring an IP address for a virtual machine
+
+You can use cloud-init to configure an IP address for a virtual machine. The IP address can be provisioned to be dynamic or static. 
+
+You can add or edit the cloud-init configuration of an existing virtual machine configuration. If you edit a running virtual machine, the virtual machine must be rebooted for the changes to take effect.
+
+.Prerequisites
+
+* DNS is configured via DHCP on the cluster.
+
+* A Linux bridge network attachment definition.
+
+* The virtual machine is connected to the bridge network. 
+
+// web console and CLI? Link to edit in web console?
+
+.Procedure
+
+. Edit or create a virtual machine configuration and include the cloud-init network details in the `spec.volumes.cloudInitNoCloud.networkData` field of the virtual machine configuration:
+
+.. To configure a dynamic IP, specify the interface name and the `dhcp` boolean:
++
+[source,yaml]
+----
+kind: VirtualMachine
+spec:
+...
+  volumes:
+  - cloudInitNoCloud:
+      networkData: |
+        version: 2
+        ethernets:
+          eth1: # or whatever interface the user wants to configure
+            dhcp4: true
+//            dhcp6: true
+----
+
+.. To configure a static IP, specify the interface name, the IP address, and the gateway address:
```

Comment:
Let's drop the gateway and limit this to IP only. The gateway configuration can have some nasty side-effects - overwrite our default gateway. Let's make this simpler for us